bin/0x0

```diff
@@ -23,14 +23,20 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+API_ENDPOINT=https://0x0.st/
+EXPIRES=0
+INPUT_ARG=""
+MIMETYPE=""
+VERBOSE=false
+
 if [[ -t 1 ]]; then
   # stdout is a TTY, enable colors
-  STYLE_COMMAND="\\e[0;97m"
-  STYLE_ARG="\\e[0;96m"
-  STYLE_ERROR="\\e[91m"
-  STYLE_BOLD="\\e[1m"
-  STYLE_NORMAL="\\e[22m"
-  STYLE_RESET="\\e[0m"
+  STYLE_COMMAND="\033[0;97m"
```

Comment:
Out of curiosity, does this change make any practical difference, or is it just stylistic?

Reply:
It turns out that `\e` doesn't work as an escape character on macos with `echo -e`, so I switched it to `\033` as suggested by this stack overflow: https://stackoverflow.com/questions/28782394/how-to-get-osx-shell-script-to-show-colors-in-echo

I tested it on linux too and it works this way.